packages/features/borrow/src/borrow-fixed/hooks/use-borrow-fixed-multi-chart.tsx

```diff
@@ -18,7 +18,7 @@ export const useBorrowFixedMultiChart = () => {
     true
   );
   const { selectedfCashId, onSelect } = useMaturitySelect('Debt', context);
-  const nToken = useNToken(deposit?.network, deposit?.currencyId);
+  const nToken = useNToken(deposit?.currencyId);
```

Comment:
We should try to just refactor this hook out of the codebase altogether, maybe look at changing the useChart hook if we're passing it in here.